app/components/exp-lookit-iframe/doc.rst

```diff
@@ -0,0 +1,68 @@
+.. _exp-lookit-iframe:
+
+exp-lookit-iframe
+==============================================
+
+Overview
+------------------
+
+A frame to display external websites, such as surveys, to the participant. 
+
+What it looks like
+~~~~~~~~~~~~~~~~~~
+
+.. image:: /../images/Exp-lookit-iframe.png
+    :alt: Example screenshot from exp-lookit-iframe frame
+
+More general functionality
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below is information specific to this particular frame. There may also be available parameters, events recorded,
+and data collected that come from the following more general sources:
+
+- the :ref:`base frame<base frame>` (things all frames do)
+
+Examples
+----------------
+
+.. code:: javascript
+
+    "study-survey": {
+        "kind": "exp-lookit-iframe",
+        "iframeSrc": "https://mit.co1.qualtrics.com/jfe/form/SV_do6kyKJkwl19USW",
```

Comment:
Maintenance consideration - will this survey example remain up indefinitely?

Reply:
Jumping in here to say that I don't think the URL needs to work since this bit of text is just meant to show an example of the frame's JSON protocol. We could replace the MIT Qualtrics link with something like `<your link here>` or `https://example.com`.

Reply:
Whoops.  I meant to replace that with something more general.  Nice catch.